core/vm/lfvm/instructions.go

```diff
@@ -23,6 +23,10 @@ func opStop(c *context) {
 
 func opRevert(c *context) {
 	//fmt.Printf("REVERT\n")
+	if c.stack.stack_ptr < 2 {
```

Comment:
nit: would be nicer to expose stack height through method instead of internal variable; something like `if c.stack.Size() < 2 {`

Reply:
done